ingest/workflow/snakemake_rules/transform.smk

```diff
@@ -67,7 +67,7 @@ rule transform:
         (cat {input.sequences_ndjson} \
             | ./bin/transform-field-names \
                 --field-map {params.field_map} \
-            | ./bin/transform-string-fields --normalize \
```

Comment:
Suggestion: delete `transform-string-fields` if it is no longer used anywhere else.

Reply:
I see, in that case a modified suggestion: remove what implements `--normalize` in `transform-string-fields`.

Reply:
Thanks, I see 2bc7409b9ab398e63aa340ee5b813efe566519e8. Would be nice to squash the two commits as one, so it's obvious why that change was made.